tests/widgets_block.rs

```diff
@@ -478,3 +480,151 @@ fn widgets_block_title_alignment_bottom() {
         Buffer::with_lines(vec!["               ", "         Title "]),
     );
 }
+
+#[test]
+fn widgets_block_multiple_titles() {
```

Comment:
We probably need more tests that show how things too long work and how multiple titles that would overlap are rendered.

Reply:
For now overlapping titles render in the following way:

┌Title1─Title2
└───────┘
The titles don't wrap to a new line, but they cut the corner a bit. My guess is this is due to the ordering of the rendering. 

I would like to make it truncate or something else, but I'm now sure how to do that.